test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_send_HB.lua

```diff
@@ -0,0 +1,98 @@
+--  Requirement summary:
+--  [Services]: SDL must be able to respond to Heartbeat_request over control service from mobile app
+--  [HeartBeat][Genivi]: SDL must track sending of HeartBeat_request from/to mobile app
+--
+--  Description:
+--  Check that no heartbeat occurs if App uses v3 protocol version and send HB to SDL in time less than HB timeout.
+--
+--  1. Used precondition
+--  SDL, HMI are running.
+--  Mobile device is connected.
+--  HeartBeatTimeout = 5000
+--
+--  2. Performed steps
+--  Start SPT, select transport, specify protocols = 3
+--  sendHeartbeatToSDL = true
+--  answerHeartbeatToSDL = false
+--  Wait 15 sec.
+--
+--  Expected behavior:
+--  1. App has successfully registered.
+--  2. App is still registered, no unexpected disconnect occurs.
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 3
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local mobile_session = require('mobile_session')
+local constants = require('protocol_handler/ford_protocol_constants')
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+commonPreconditions:BackupFile("smartDeviceLink.ini")
+
+function Test:StartSDL_And_Connect_Mobile()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:Start_Session_And_Register_App()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  commonFunctions:write_parameter_to_smart_device_link_ini("HeartBeatTimeout", 5000)
```

Comment:
@LitvinenkoIra, update of INI file should be done before SDL starts, because SDL loads information of it at start up and after that will not apply changes.